android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandler.java

```diff
@@ -35,6 +35,7 @@
   private float mHitSlop[];
 
   private boolean mShouldCancelWhenOutside;
+  private int mNumberOfPointers = 1;
```

Comment:
Doesn't matter too much but 0 would be a more appropriate initial value as there are no pointers when handler gets instantiated